sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,3 @@
+select *
+  from "customers"
+  where "isActive" = 'false';
```

Comment:
We don't normally quote booleans. This works because Postgres does aggressive data type conversion.